packages/tools/e2e-tests/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@kadena-dev/e2e-tests",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "format": "pnpm run --sequential /^format:.*/",
+    "format:lint": "pnpm run lint:src --fix",
+    "format:md": "remark README.md -o --use @kadena-dev/markdown",
+    "format:src": "prettier . --cache --write",
+    "install:chromium": "npx playwright install chromium",
+    "lint": "pnpm run /^lint:.*/",
+    "lint:fmt": "prettier . --cache --check",
+    "lint:pkg": "lint-package",
+    "lint:src": "eslint src --ext .js,.ts",
+    "test:e2e": "npx playwright test"
```

Comment:
I think you can omit `npx`, it should be available as a binary in `$PATH`. Package managers add `node_modules/.bin` to `$PATH` automatically.